MultimodalRAGWithVideos/ui/gradio/mmragwithvideos_ui_gradio.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
why is this file empty?

Reply:
@Spycsh  I have pushed this file. Please review. Thanks

Reply:
This is not empty anymore. We implemented UI for this app.